lib/elixir/test/elixir/kernel/overridable_test.exs

```diff
@@ -214,4 +216,20 @@ defmodule Kernel.OverridableTest do
       """
     end
   end
+
+  test "overriding private functions is deprecated and emits a warning" do
```

Comment:
Yeah, maybe 👍 on skipping this test. I'll update if we decide to go with this deprecation.